controllers/internal/hostdefiner/validator.go

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright 2022 IBM Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package hostdefiner
+
+// Validate checks if the spec is valid
+// Replace it with kubernetes native default setter when it is available.
+// https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/#validation
+func (c *HostDefiner) Validate() error {
+	return nil
```

Comment:
do we really need this?

Reply:
I copied it from CSI controller, why did we have it in the first place?

Reply:
done